sdcflows/cli/find_estimators.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from pathlib import Path
+
+
+def _drop_sub(value):
+    return value[4:] if value.startswith("sub-") else value
+
+
+def _parser():
+    parser = argparse.ArgumentParser(
+        description="Parse a BIDS directory and show what estimators are available",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    parser.add_argument("bids_dir", type=Path, help="The input BIDS directory to parse")
+    parser.add_argument(
+        "-s",
+        "--subjects",
+        type=_drop_sub,
+        nargs="+",
+        help="One or more subject identifiers",
+    )
+    parser.add_argument(
+        "--fmapless",
+        action="store_true",
+        default=False,
+        help="Allow fieldmap-less estimation",
+    )
+    parser.add_argument(
+        "--bids-database-dir",
+        metavar="PATH",
+        type=Path,
+        help="Path to a PyBIDS database folder, for faster indexing (especially "
+             "useful for large datasets). Will be created if not present."
+    )
+    return parser
+
+
+def gen_layout(bids_dir, database_dir=None):
+    import re
+    from bids.layout import BIDSLayout, BIDSLayoutIndexer
+
+    _indexer = BIDSLayoutIndexer(
+        validate=False,
+        ignore=(
+            "code",
+            "stimuli",
+            "sourcedata",
+            "models",
+            "derivatives",
+            re.compile(r"^\."),
+            re.compile(r"sub-[a-zA-Z0-9]+(/ses-[a-zA-Z0-9]+)?/(anat|beh|eeg|ieeg|meg|micr|perf)"),
```

Comment:
Realizing we're going to need anat for SyN.

```suggestion
            re.compile(r"sub-[a-zA-Z0-9]+(/ses-[a-zA-Z0-9]+)?/(beh|eeg|ieeg|meg|micr|perf)"),
```